selenium-system-test/src/test/scala/hts/pages/WebPage.scala

```diff
@@ -34,7 +34,24 @@ trait WebPage extends Matchers
 
   def back()(implicit driver: WebDriver): Unit = clickOn("ButtonBack")
 
-  def nextPage()(implicit driver: WebDriver): Unit = find(CssSelectorQuery(".page-nav__link.page-nav__link--next")).foreach(click.on)
+  def nextPage()(implicit driver: WebDriver): Unit = {
+    driver.findElement(By.id("get-help")).sendKeys(Keys.chord(Keys.CONTROL, Keys.END))
+    find(CssSelectorQuery(".page-nav__link.page-nav__link--next")).foreach(click.on)
+  }
+
+  /*
+   * CONFIRM CURRENT PAGE INFO
+   */
+
+  def expectedUrl: Option[String]
```

Comment:
Why is the return type `Option[String]`? Does `None` mean checks against this expected URL should always return `true`? The way you have overriden it, this seems to always return either `Some` with a value in it or `Some` with an empty `String` in it. Is the `Some("")` actually your `None` case?

Reply:
Hi Andy,

I set it as Option[String] so that I could set them as None in the event that the page or element to which the comparison was taking place wouldn't fail if that particular object didn't have those elements (admittedly that's probably not necessary for URL as that will always be present). I'm still pretty new to Scala however, so I'm open to any suggestions or corrections. 

The ones currently just set to Some("") are pages which never use the pageInfoIsCorrect method. In future I am of the position that these details should always be checked when a new page is loaded.